kafka-rest/src/main/java/io/confluent/kafkarest/entities/v3/TopicConfigData.java

```diff
@@ -21,19 +21,19 @@
 import javax.annotation.Nullable;
 
 /**
- * A (Topic) configuration resource type.
+ * A (Topic) config resource type.
  */
-public final class TopicConfigurationData {
+public final class TopicConfigData {
 
-  public static final String ELEMENT_TYPE = "configuration";
+  public static final String ELEMENT_TYPE = "config";
```

Comment:
is this what ends up in the CRN? possibly consider `config-property` as more precise.